src/schema/v2/partner/Settings/deletePartnerLocationMutation.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { LocationType } from "schema/v2/location"
+import { ResolverContext } from "types/graphql"
+
+interface DeletePartnerLocationInputProps {
+  LocationId: string
+  partnerId: string
+}
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "DeletePartnerLocationSuccess",
+  isTypeOf: (data) => !!data.id,
+  fields: () => ({
+    location: {
```

Comment:
A bit of drift here, all location related mutations nest under `location` where contact related mutations use `partnerContact`

Can adjust `partnerContact` to be `contact` if thats a cleaner read

Reply:
In general you can add anything here that makes sense - even `partner` etc. (if an entire subgraph needs to be re-fetched).